packages/meditrak-server/package.json

```diff
@@ -47,7 +47,7 @@
     "form-data": "^2.3.3",
     "format-link-header": "^2.1.0",
     "http": "^0.0.0",
-    "lodash": "^4.17.11",
```

Comment:
So it looks like `lodash` wasn't being used anywhere in `meditrak-server`?

Reply:
Nice!